SynonymCreator/src/main/scala/no/nrk/samnorsk/wikiextractor/TranslationCounter.scala

```diff
@@ -0,0 +1,64 @@
+package no.nrk.samnorsk.wikiextractor
+
+import scala.collection.mutable
+
+class TranslationCounter[A, B](sourceTfFilter: Int = 1, sourceDfFilter: Double = 1.0,
+                               transTfFilter: Int = 1, transDfFilter: Double = 1.0,
+                               topN: Option[Int] = Option.empty) {
+  private val translations = mutable.HashMap[A, mutable.Map[B, Int]]()
```

Comment:
Min versjon "streamer" par til telleren, mens din teller opp underveis og slår sammen deltellingene?

Vi sette dette sammen in neste runde så det blir felles struktur.